### ISSUE TITLE
Feature Request: Show debugging information for SQL queries

### ISSUE THREAD

**Issue by ChristopherBass** [Kentico/xperience-by-kentico-miniprofiler#10]
A very useful feature Kentico 13 had was SQL debugging - the ability to review a page load and see what specific SQL queries were run, and how long they took (including highlighting long-running queries). 

This helped with determining if our caching was working (if the queries don't run and the data still shows up, the cache succeeded) - it also helped with performance-tuning queries and detecting when devs didn't apply proper filters/paging to queries (by viewing the result count).

Miniprofiler appears to be able to do that, judging by https://miniprofiler.com/dotnet/ and  https://docs.servicestack.net/built-in-profiling#sql-profiling (The latter is a third-party service and is using the MVC (not core) version of Miniprofiler, but both links show screenshots of Miniprofiler being used in the way I'm imagining)

Currently I don't believe the Kentico Miniprofiler implementation (Or XbyK internally) gives any insight into SQL queries at all, so anything would be an improvement.

**Comment by liparova** [comment]
Thank you for highlighting this request. We will review it internally and keep you informed of any developments.

**Comment by seangwright** [comment]
@ChristopherBass are you sure you tried out this integration?

Here's what it looks like when enabled in the [Kentico Community Portal](https://github.com/kentico/community-portal)

![image](https://github.com/Kentico/xperience-by-kentico-miniprofiler/assets/1382768/b6d907e5-f7c1-4135-8629-8e86003e166f)

If you click on the "sql" column for the one of the results, it will show all the SQL queries run within that part of the call stack.

![image](https://github.com/Kentico/xperience-by-kentico-miniprofiler/assets/1382768/4f422dc3-a2f8-4eeb-98c8-bddfe8398c53)

**Comment by ChristopherBass** [comment]
I also reproduced this in 29.1.3 on Dancing Goat (updated to .net 8)
[Program.zip](https://github.com/user-attachments/files/15774647/Program.zip)
![2024-06-10 10_18_16-Home - Dancing Goat](https://github.com/Kentico/xperience-by-kentico-miniprofiler/assets/5831169/987106f2-5290-4359-9515-b178691d1568)

**Comment by ChristopherBass** [comment]
Comparing projects with Sean Wright, I think this is just a documentation fix.

The documentation just says to put the 'AddKenticoMiniProfiler' call in your program.cs

and gives this as guidance:

```
var env = builder.Environment;

if (env.IsDevelopment())
{
    builder.Services.AddKenticoMiniProfiler();
}

// Other services ...
```

But if you put it before the builder.Services.AddKentico(), then you get the non-SQL version. It's possible AddKentico is touching the sql in a way that's kicking the profiler off of the SqlConnection, but in either case, as long as you add it after AddKentico() it seems to work as designed.

**Comment by seangwright** [comment]
I think this was resolved by #11. We can open this issue back up if there are any additional problems related to the dependency registration/docs.
